Backends/Null/port.lisp

```diff
@@ -27,8 +27,8 @@
 
 (defclass null-port (basic-port)
   ((id)
-   (pointer :accessor port-pointer :initform (make-instance 'null-pointer))
-   (window :initform nil :accessor null-port-window)))
+   (window :initform nil :accessor null-port-window))
+  (:default-initargs :pointer (make-instance 'null-pointer)))
```

Comment:
what is this null-port-window? I don't see any mention in the commit message about it, nor it is referenced elsewhere in this commit.

Reply:
I don't know what is `null-port-window`, it  was already there I didn't add it, I only removed `pointer` slot because I add it to `basic-port`. 
I think it is there because clx port have a `clx-port-window` where it is stored the xwindow of the root.
Anyway if you want we can remove it, it is never used and it is not in the specification.

Reply:
right, I've misread the diff - sorry.